### ISSUE TITLE
Persistent history after closing

### ISSUE THREAD

**Issue by LamprosPitsillos** [nakst/gf#89]
Is there a way to set history to be persistent ?
Something like `.gdb_history` or  `set history save on`

**Comment by nakst** [comment]
History of what?

**Comment by LamprosPitsillos** [comment]
Of the commands executed in the debug window
```
p x
set var x=4
b myfunc()
__restart_gf__
__press_up__
b myfunc()
...
```
gdb does this using `.gdb_history` that you enable with `set history save on` if i an not mistaken

**Comment by nakst** [comment]
No, this is no currently possible. However, it would not be too difficult to add, if you'd like to make a pull request. Basically, you need to the same thing that `restoreWatchWindow` does, but with the command history array, `Array<char *> commandHistory;`.

**Comment by LamprosPitsillos** [comment]
I will give it a try after my exams !
